src/main/java/com/github/underscore/U.java

```diff
@@ -621,6 +621,34 @@ public boolean contains(final T elem) {
         return contains(list.subList(fromIndex, list.size()), elem);
     }
 
+    public boolean containsAtLeast(final Integer count, final T value) {
```

Comment:
It may be `iterable, value, count` parameters order and `count` type is `int` not `Integer`.